x-pack/plugins/siem/public/common/components/endpoint/page_view.tsx

```diff
@@ -74,69 +80,94 @@ export const PageViewBodyHeaderTitle = memo<{ children: ReactNode }>(
-     */
-    bodyHeader?: ReactNode;
-    children?: ReactNode;
-  }
->(({ viewType, children, headerLeft, headerRight, bodyHeader, ...otherProps }) => {
-  return (
-    <StyledEuiPage
-      className={(viewType === 'list' && 'endpoint--isListView') || 'endpoint--isDetailsView'}
-      {...otherProps}
-    >
-      <EuiPageBody>
-        {(headerLeft || headerRight) && (
-          <EuiPageHeader className="endpoint-header">
-            <EuiPageHeaderSection data-test-subj="pageViewHeaderLeft">
-              {isStringOrNumber.test(typeof headerLeft) ? (
-                <PageViewHeaderTitle>{headerLeft}</PageViewHeaderTitle>
-              ) : (
-                headerLeft
-              )}
-            </EuiPageHeaderSection>
-            {headerRight && (
-              <EuiPageHeaderSection data-test-subj="pageViewHeaderRight">
-                {headerRight}
-              </EuiPageHeaderSection>
-            )}
-          </EuiPageHeader>
-        )}
-        <EuiPageContent className="endpoint-page-content">
-          {bodyHeader && (
-            <EuiPageContentHeader>
-              <EuiPageContentHeaderSection data-test-subj="pageViewBodyTitleArea">
-                {isStringOrNumber.test(typeof bodyHeader) ? (
-                  <PageViewBodyHeaderTitle>{bodyHeader}</PageViewBodyHeaderTitle>
+export const PageView = memo<PageViewProps>(
+  ({ viewType, children, headerLeft, headerRight, bodyHeader, tabs, ...otherProps }) => {
+    const tabComponents = useMemo(() => {
+      if (!tabs) {
+        return [];
+      }
+      return tabs.map(({ name, id, ...otherEuiTabProps }) => (
+        <EuiTab {...otherEuiTabProps} key={id}>
+          {name}
+        </EuiTab>
+      ));
+    }, [tabs]);
+
+    return (
+      <StyledEuiPage
+        className={(viewType === 'list' && 'endpoint--isListView') || 'endpoint--isDetailsView'}
+        {...otherProps}
+      >
+        <EuiPageBody>
+          {(headerLeft || headerRight) && (
+            <EuiPageHeader className="endpoint-header">
+              <EuiPageHeaderSection data-test-subj="pageViewHeaderLeft">
+                {isStringOrNumber.test(typeof headerLeft) ? (
+                  <PageViewHeaderTitle>{headerLeft}</PageViewHeaderTitle>
                 ) : (
-                  bodyHeader
+                  headerLeft
                 )}
-              </EuiPageContentHeaderSection>
-            </EuiPageContentHeader>
+              </EuiPageHeaderSection>
+              {headerRight && (
+                <EuiPageHeaderSection data-test-subj="pageViewHeaderRight">
+                  {headerRight}
+                </EuiPageHeaderSection>
+              )}
+            </EuiPageHeader>
           )}
-          <EuiPageContentBody data-test-subj="pageViewBodyContent">{children}</EuiPageContentBody>
-        </EuiPageContent>
-      </EuiPageBody>
-    </StyledEuiPage>
-  );
-});
+          {tabs && <EuiTabs className="endpoint-navTabs">{tabComponents}</EuiTabs>}
```

Comment:
nit -> {tabComponents.lenght > 0 && <EuiTabs className="endpoint-navTabs">{tabComponents}</EuiTabs>}

Reply:
👍 thanks.

Reply:
@XavierM will make this change in my next PR since its minor.